design/ceph/ceph-config-via-cluster-crd.md

```diff
@@ -51,3 +51,5 @@ The operator will use the Ceph config store (that is accessed via `ceph config a
 ### Limitations
 
 The operator won't be unsetting any previously set config options or restore config options to their default value.
+
+The operator will ignore a few vital config options, such as `fsid`, `keyring` and `mon_host`. This list can be extended later on to add options that cause issues/conflicts with the features in the operator.
```

Comment:
I'd suggest to keep it simple, no need to tell them we can make changes. Hopefully that's understood for anything in the repo.
```suggestion
The operator will ignore a few vital config options, such as `fsid`, `keyring` and `mon_host`.
```

Reply:
I have updated it.